### ISSUE TITLE
Change alt text for Westside Expunge Assist image on events page

### ISSUE THREAD

**Issue by blulady** [hackforla/website#3316]
### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). For this issue, we will change the alt text for the Westside Expunge Assist location on [Events page](https://www.hackforla.org/events/).

### Details
Designers and PMs identified that some alt texts in the Events page should be changed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items
- [x] For the file `_includes/events-page/our-locations-content.html`, change the alt text so that
```
<img id="mini-img" src="../assets/images/projects/expunge-assist.png" alt="Expunge Assist" />
```
becomes
```
<img id="mini-img" src="../assets/images/projects/expunge-assist.png" alt="" />
```
- [ ] **For merge team**: Once the pull request associated with this issue is approved and merged, please update and edit epic #3132 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #3132 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is part of epic #3132 
- [Events page](https://www.hackforla.org/events/)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

#### File and code links you will need to work on this issue
- Directory to find the page once you have it in your IDE: `_includes/events-page/our-locations-content.html`
- Click on the link to see the source file code: https://github.com/hackforla/website/blob/gh-pages/_includes/events-page/our-locations-content.html

#### Information about WCAG
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)

**Comment by 7kram** [comment]
@ExperimentsInHonesty Hey I'm still seeing the origin alt text. Refer to the screenshot:
<img width="468" alt="Screen Shot 2022-09-28 at 8 20 03 AM" src="https://user-images.githubusercontent.com/107599650/192818762-658c60d8-c1a5-4632-bc5f-52f2b6fb9b73.png">

This issue should be reviewed by a dev lead and returned to the prioritized backlog if necessary.

**Comment by JessicaLucindaCheng** [comment]
Reopening because the alt text has not been changed as requested in the issue.

**Comment by wendybarrios** [comment]
Availability for this week: [Tuesday/Thursday/Friday]: (9:00 am - 12:00 pm and 5:30 pm - 7:00pm)
My estimated ETA for completing this issue: Tuesday, November 29
